src/test/java/org/junit/tests/running/methods/TestMethodTest.java

```diff
@@ -251,5 +251,29 @@ public void onlyIgnoredMethodsIsStillFineTestClass() {
         Result result = JUnitCore.runClasses(OnlyTestIsIgnored.class);
         assertEquals(0, result.getFailureCount());
         assertEquals(1, result.getIgnoreCount());
+        assertEquals(0, result.getRunCount());
     }
+
+    public static class OneTestIsIgnoredAndOneIsIncomplete {
+        @BeforeClass
+        public static void throwException() throws Exception {
```

Comment:
Could you add another test where the exception is thrown by an `@AfterClass` method?